### PR TITLE
WINC-1182: [node] Modify filesystem queries to include Windows metrics

### PR DESCRIPTION
**Pull request by mansikulkarni96**
This commit modifies filesystem queries to account for Windows node metrics which are exported through windows_exporter when Windows nodes are present in a cluster.

### CODE REVIEW

**Comment by TheRealJon** [comment]
@mansikulkarni96 Is there a Jira issue related to this change? We'll need a bug or RFE to merge this.

**Comment by mansikulkarni96** [comment]
@TheRealJon I linked the issue we have on the Windows Containers JIRA board.

**Comment by mtnbikenc** [comment]
@TheRealJon What else is needed here to get this reviewed and merged?

**Comment by TheRealJon** [comment]
A review from the monitoring team would be best. They have much more expertise in PromQL than I do. I can tag and approve it once they have reviewed.

File: frontend/packages/console-app/src/components/nodes/NodesPage.tsx
Comment (rexagod) [inline frontend/packages/console-app/src/components/nodes/NodesPage.tsx:408]:
```suggestion
        'sum by (instance) ((max by (device, instance) (node_filesystem_size_bytes{device=~"/.*"})) - (max by (device, instance) (node_filesystem_free_bytes{device=~"/.*"}))) + ' +
        'sum by (instance) ((max by (volume, instance) (windows_logical_disk_size_bytes)) - (max by (volume, instance) (windows_logical_disk_free_bytes)))',
```

Assuming you want the total used storage from **both** windows and linux nodes, I believe this should add the two, since doing an OR will only `group_left` and return the left value for the same labelset.

Comment (rexagod) [reply frontend/packages/console-app/src/components/nodes/NodesPage.tsx:408]: PS. I don't know what this query is for exactly in UI, I'm basing my understanding from the earlier discussion and the context around this code snippet (`usedStorage`), so feel free to correct me if I missed something here. :)

Comment (mansikulkarni96) [reply frontend/packages/console-app/src/components/nodes/NodesPage.tsx]: @rexagod  The intent is to return what is available from these two queries. If it is executed for a Windows node only the windows part of the query will return a value. This query is used on the individual node page of each of the nodes present in the cluster to display usage graphs.

Comment (rexagod) [reply frontend/packages/console-app/src/components/nodes/NodesPage.tsx:408]: Ah, in that case, for a per-node basis, `or` should be the best choice. 👍🏼

Comment (rexagod) [reply frontend/packages/console-app/src/components/nodes/NodesPage.tsx:408]: I didn't notice this earlier but these are `node_exporter` metrics, hence these will always be exposed on a per-node basis, i.e., to be explicit, `or` should be the right choice here.

**Comment by rexagod** [comment]
/lgtm

Monitoring ACK!

**Comment by rexagod** [comment]
/hold 

Holding for others. Feel free to /unhold.

**Comment by mansikulkarni96** [comment]
@TheRealJon can you please take a look and approve this if it looks good?

**Review by TheRealJon — APPROVED** [review]
/unhold
/lgtm

**Pull request merged**
